docs/sugar/05-upload-methods.md

```diff
@@ -0,0 +1,24 @@
+# Upload Methods
+
+There are currently two upload (storage) methods available in Sugar: `"bundlr"` and `"aws"`.
+
```

Comment:
Add this sentence:

Uploading in Sugar is designed as a Rust trait to make it easier to for new methods to be added. We plan to release a developer guide for how to do this so we can support a wide-variety of upload methods.